docs/content/entity-templates.md

```diff
@@ -97,9 +97,9 @@ public static class CubeTemplate
 
 ### Player lifecycle module
 
-The `AddPlayerLifecycleComponents` extension method for this Feature Module adds the `PlayerHeartbeatClient` and `PlayerHeartbeatServer` components to an entity. It requires the [worker attributes](./glossary.md#worker-attributes) of the client, `clientAttribute`, and the server, `serverAttribute`.
+The `AddPlayerLifecycleComponents` extension method for this Feature Module adds the `PlayerHeartbeatClient` and `PlayerHeartbeatServer` components to an entity. It requires the [worker attributes]({{urlRoot}}/content/glossary#worker-attribute) of the client, `clientAttribute`, and the server, `serverAttribute`.
 
-The following code snippet shows how to pass  [worker attributes](./glossary.md#worker-attributes) into the `AddPlayerLifecycleComponents` extension method. The `workerId` is the ID of the worker that sent a player creation request, and `clientAttributeSet` is the full list of attributes of this worker. Both fields are populated by the `HandleCreatePlayerRequestSystem.cs` system within the player lifecycle module.
+The following code snippet shows how to pass  [worker attributes]({{urlRoot}}/content/glossary#worker-attribute) into the `AddPlayerLifecycleComponents` extension method. The `workerId` is the ID of the worker that sent a player creation request, and `clientAttributeSet` is the full list of attributes of this worker. Both fields are populated by the `HandleCreatePlayerRequestSystem.cs` system within the player lifecycle module.
 
 **Example**<br/>
```

Comment:
`<br/>` 😡

Reply:
That style is used in lots of other places, so I'll leave it for now.

Reply:
I know - just makes me sad